module/account/src/Application/Validator/Constraints/AvailableTokenConstraintValidator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Account\Application\Validator\Constraints;
+
+use Ergonode\Account\Domain\Repository\UserResetPasswordTokenRepositoryInterface;
+use Ergonode\Account\Domain\ValueObject\ResetToken;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+class AvailableTokenConstraintValidator extends ConstraintValidator
```

Comment:
it doesn't have to be Symfony validator anymore therefore constraints as arguments become redundant